eta/models/manifest.json

```diff
@@ -373,11 +373,11 @@
             "requirements": {
                 "cpu": {
                     "support": true,
-                    "packages": ["tensorflow<2"]
+                    "packages": ["tensorflow"]
                 },
                 "gpu": {
                     "support": true,
-                    "packages": ["tensorflow-gpu<2"]
+                    "packages": ["tensorflow-gpu"]
```

Comment:
Does `tensorflow-gpu` work for version `>=2`? The official docs say `tensorflow>=2` will include GPU support, but the problem is that these requirements should technically read `tensorflow-gpu<2 OR tensorflow>=2`, but I'm not sure if `pip` allows package specifications with `OR` in them, and I don't really want to write our own code for that...

Reply:
You can `pip install tensorflow-gpu==2` so it should work, no?

Reply:
But users are likely to have already installed TF 2.X via `pip install --upgrade tensorflow` like the official docs recommend. As is, they'd get a "`tensorflow-gpu` not installed" error here. Although they could set `error_level` to ignore it, I'd rather our code not force people to use workarounds when following the "official" way to use other packages.

I'm going to tweak our package verification code to allow the `<requirement1>|<requirement2>` syntax